pyrogram/handlers/error_handler.py

```diff
@@ -35,5 +33,9 @@ class ErrorHandler(Handler):
             The received error.
     """
 
+    # TODO ⬇️
+    """For a nicer way to register this handler, have a look at the
+    :meth:`~pyrogram.Client.on_error` decorator."""
```

Comment:
ok, it's good to have an easiest way to handle errors.

Reply:
complete